modules/porous_flow/include/materials/PorousFlowPermeabilityConst.h

```diff
@@ -0,0 +1,39 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#ifndef POROUSFLOWPERMEABILITYCONST_H
+#define POROUSFLOWPERMEABILITYCONST_H
+
+#include "DerivativeMaterialInterface.h"
+#include "Material.h"
```

Comment:
The Material.h, PorousFlowDictator.h and Deriv...h includes are probaly unnecessary as they'll come with PorousFlowPermeabilityUnity.h